provider/gateway/rest/router.go

```diff
@@ -391,6 +400,25 @@ func leaseShellHandler(log log.Logger, mclient pmanifest.Client, cclient cluster
 	}
 }
 
+type versionInfo struct {
+	Akash string            `json:"akash"`
```

Comment:
should we include commit hash, build date, go version, platform, etc...?

Reply:
Added all the useful information that is available via `akash version` command.